index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fitness</title>
+    <link rel="stylesheet" href="main.css" />
+    <script
+      src="https://kit.fontawesome.com/667d7efdcc.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <div class="desk_nav">
+        <div class="list-1">
+          <ul>
+            <i class="fa-brands fa-facebook"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <li><a href="#">Logout</a></li>
+            <li><a href="#">Mypage</a></li>
+            <li><a href="#">Somali</a></li>
+          </ul>
+        </div>
+        <nav class="nav">
+          <div lgo>
+            <a href="index.html">
+              <img src="./img/logo.png" alt="" class="logo"
+            /></a>
+          </div>
+          <ul>
+            <a href="./about.html">About</a>
+            <a href="#">Program</a>
+            <a href="#">Join</a>
+            <a href="#">Sponsor</a>
+            <a href="#">Updates</a>
+            <h4 class="list-h4">BB compaign</h4>
+          </ul>
+        </nav>
+      </div>
+      <div class="mobile-menu">
+        <li class="menuBtn">
+          <i class="fa-solid fa-bars fa-2x"></i>
+        </li>
+        <div class="menu hidden">
+          <ul>
+            <a href="./about.html">About</a>
+            <a href="#">Program</a>
+            <a href="#">Join us</a>
+            <a href="#">Sponsor</a>
+            <a href="#">Updates</a>
+          </ul>
+        </div>
+        <li class="close-menu hidden"><i class="fa-solid fa-x fa-2x"></i></li>
+      </div>
+      <div class="description">
+        <h3 class="desc-header">"Hello! body lovers"</h3>
+        <h1>
+          Making your body smoothier? <br />
+          here is the dream seminar.
+        </h1>
+        <p class="p-1">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere
+          dolorum impedit recusandae ullam alias
+        </p>
+        <h3 class="time">2023.04.25(Tuesday) - 28(Friday)</h3>
+        <p class="p-2 venue">@ GetPo central station, 5th street</p>
+      </div>
```

Comment:
- [ ] Make sure to remove this section in the `header` section. you may follow this form. Kindly fix it 👍 . 

````
<body>
       <header>
             <a>logo</a>
             <nav></nav>
       </header>
       <section>
             ........
       </section>
       <section>
             ........
       </section>

<body>
```` 

- [ ] Also make sure to follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
![Capture2](https://user-images.githubusercontent.com/57408419/234551706-54f8dc82-8206-422c-be01-90aaacf043ab.PNG)
> There is a `background-clip` on your h1
````
 background-image: url('...link to the image...');
  background-size: cover;
  -webkit-background-clip: text;
````

  - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements.

#### Template design
![Capture1](https://user-images.githubusercontent.com/57408419/234552087-26e0a184-b768-4aa5-9fad-6cf2bd06bf3e.PNG)

##### Your design

![Capture](https://user-images.githubusercontent.com/57408419/234552229-d2de4019-5eee-48cb-8260-cbe8fa40e301.PNG)

*NOTE*: Make sure you did the same for the `about.html` file 👍